### PR TITLE
Category text

### PR DESCRIPTION
**Pull request by valhristov**
Fix SonarSource/sonarlint-visualstudio#434

### CODE REVIEW

File: scripts/rspec/rspec.ps1 [hunk 1]
```diff
@@ -56,9 +56,9 @@ if (-Not (Test-Path $resgenPath)) {
...
 $categoriesMap = @{
-    "BUG" = "Sonar Bug";
-    "CODE_SMELL" = "Sonar Code Smell";
-    "VULNERABILITY" = "Sonar Vulnerability";
+    "BUG" = "Bug";
+    "CODE_SMELL" = "Code Smell";
+    "VULNERABILITY" = "Vulnerability";
 }
 
 $severitiesMap = @{
```
Comment (michalb-sonar) [inline scripts/rspec/rspec.ps1]: What does this map do?

Comment (valhristov) [reply scripts/rspec/rspec.ps1]: Converts the SonarQube categories to displayable strings.

File: sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/RspecStrings.resx [hunk 2]
```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="S100_Category" xml:space="preserve">
-    <value>Sonar Code Smell</value>
+    <value>Minor Code Smell</value>
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/RspecStrings.resx]: Have you also updated all VB rules?

Comment (valhristov) [reply sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/RspecStrings.resx]: Yes, it is in the same commit

**Pull request merged**